Updated README file (#27)
wagtail/guide: README.md

# Guide

Guide is a website to help content editors, moderators, administrators, and other users learn how to user the Wagtail content management system (CMS).

The Wagtail guide will ultimately include:
- Tutorials 
- How-to articles
- Reference materials

> - Walkthroughs and visual learning materials

Reviewer: This list was about the sections/modes we adopt. Isn't it better to stay close to the original writing modes?

- Tutorials 
- How-to
- Explanation
- Reference

We could explain a bit more: https://documentation.divio.com/

> The Wagtail guide will ultimately include:
> - Tutorials 
> - How-to articles
> - Reference materials
> - Walkthroughs and visual learning materials

Reviewer: ```suggestion
The Wagtail guide will ultimately include:
- Tutorials
- How-to
- Explanation
- Reference

Learn more about the [documentation system](https://documentation.divio.com/).
```

Author: So I added nouns "articles"/"materials" to "How-To" and "Reference" because "How-to" is a compound adjective that needs a noun to make the grammar work and the singular version of "Reference" would be incorrect for the meaning we're aiming for. The plural "References" also doesn't work because when most English-speaking people think of "references", they think of citations or job "references". If you look in Divio's copy, they do a similar thing. The singular "Explanation" also doesn't work because it implies the guide will only contain a single explanation. So I provided examples of what those explanations would be instead to give people a better idea what it would contain.

I'm going to keep the phrasing, but I think the addition of the link is a good idea.

# Table of Contents

- [Installation](#installation)
- [Gitpod](#gitpod)
- [Contributing](#contributing)
- [Other Notes](#other-notes)

# Installation

We assume that you have basic knowledge of Node/Yarn/Webpack and Python/Django/Wagtail in these instructions. We recommend you develop Wagtail Guide locally on your machine using venv.

#### Dependencies

> - Python 3.9

Reviewer 2: or Python 3.10?

- Git
- Node 16.*
- [Yarn](https://yarnpkg.com/)

### Setting up Wagtail guide in a virtual environment

> With [PIP](https://github.com/pypa/pip) installed, run:

Reviewer: AFAIK Pip comes preinstalled with modern Python.

I think we can drop this line.

Author: Good catch.

    python -V

> Confirm that the output is showing version Python 3.9. If not, you may have multiple versions of python installed on your system and will need to switch to the appropriate version when creating the virtual environment.

Reviewer 2: ```suggestion
Confirm that the output is showing version Python 3.9. If not, you may have multiple versions of Python installed on your system and will need to switch to the appropriate version when creating the virtual environment.
```
```suggestion
Confirm that the output is showing version Python 3.9. If not, you may have multiple versions of python installed on your system and will need to switch to the appropriate version when creating the virtual environment.
```

1. Click the ``Open in Gitpod`` button.
2. Relax: a development environment with an active Wagtail Guide site will be created for you.
3. Login at `/admin/` with username `admin` and password `changeme`

# Contributing

> If you're a Python or Django developer, fork the repo and join us. You'll find answers to many common new contributor questions in our [contributing guidelines](https://github.com/wagtail/bakerydemo/blob/master/contributing.md).

Reviewer: Why do we link to the contributing guide of bakerydemo?

We should add `CONTRIBUTING.md` to this project.

Author: Because apparently I didn't update the link to the main Wagtail contributing guidelines when I copied the README structure over that I liked from the bakerydemo. We can definitely add a CONTRIBUTING.md.
